<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeroulementCryptSub</class>
 <widget class="QWidget" name="DeroulementCryptSub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeroulementCryptSub</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>110</y>
     <width>461</width>
     <height>301</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>tabText</string>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>28</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>15</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>21</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>15</number>
   </attribute>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>110</y>
     <width>381</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>41</y>
     <width>846</width>
     <height>62</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>tabKey</string>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <row>
    <property name="text">
     <string>Clair</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Chiffr√©</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nouvelle colonne</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>A</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>B</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>C</string>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>D</string>
    </property>
   </item>
   <item row="0" column="4">
    <property name="text">
     <string>E</string>
    </property>
   </item>
   <item row="0" column="5">
    <property name="text">
     <string>F</string>
    </property>
   </item>
   <item row="0" column="6">
    <property name="text">
     <string>G</string>
    </property>
   </item>
   <item row="0" column="7">
    <property name="text">
     <string>H</string>
    </property>
   </item>
   <item row="0" column="8">
    <property name="text">
     <string>I</string>
    </property>
   </item>
   <item row="0" column="9">
    <property name="text">
     <string>J</string>
    </property>
   </item>
   <item row="0" column="10">
    <property name="text">
     <string>K</string>
    </property>
   </item>
   <item row="0" column="11">
    <property name="text">
     <string>L</string>
    </property>
   </item>
   <item row="0" column="12">
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item row="0" column="13">
    <property name="text">
     <string>N</string>
    </property>
   </item>
   <item row="0" column="14">
    <property name="text">
     <string>O</string>
    </property>
   </item>
   <item row="0" column="15">
    <property name="text">
     <string>P</string>
    </property>
   </item>
   <item row="0" column="16">
    <property name="text">
     <string>Q</string>
    </property>
   </item>
   <item row="0" column="17">
    <property name="text">
     <string>R</string>
    </property>
   </item>
   <item row="0" column="18">
    <property name="text">
     <string>S</string>
    </property>
   </item>
   <item row="0" column="19">
    <property name="text">
     <string>T</string>
    </property>
   </item>
   <item row="0" column="20">
    <property name="text">
     <string>U</string>
    </property>
   </item>
   <item row="0" column="21">
    <property name="text">
     <string>V</string>
    </property>
   </item>
   <item row="0" column="22">
    <property name="text">
     <string>W</string>
    </property>
   </item>
   <item row="0" column="23">
    <property name="text">
     <string>X</string>
    </property>
   </item>
   <item row="0" column="24">
    <property name="text">
     <string>Y</string>
    </property>
   </item>
   <item row="0" column="25">
    <property name="text">
     <string>Z</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>Z</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>Y</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>X</string>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string>W</string>
    </property>
   </item>
   <item row="1" column="4">
    <property name="text">
     <string>V</string>
    </property>
   </item>
   <item row="1" column="5">
    <property name="text">
     <string>U</string>
    </property>
   </item>
   <item row="1" column="6">
    <property name="text">
     <string>T</string>
    </property>
   </item>
   <item row="1" column="7">
    <property name="text">
     <string>S</string>
    </property>
   </item>
   <item row="1" column="8">
    <property name="text">
     <string>R</string>
    </property>
   </item>
   <item row="1" column="9">
    <property name="text">
     <string>Q</string>
    </property>
   </item>
   <item row="1" column="10">
    <property name="text">
     <string>P</string>
    </property>
   </item>
   <item row="1" column="11">
    <property name="text">
     <string>O</string>
    </property>
   </item>
   <item row="1" column="12">
    <property name="text">
     <string>N</string>
    </property>
   </item>
   <item row="1" column="13">
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item row="1" column="14">
    <property name="text">
     <string>L</string>
    </property>
   </item>
   <item row="1" column="15">
    <property name="text">
     <string>K</string>
    </property>
   </item>
   <item row="1" column="16">
    <property name="text">
     <string>J</string>
    </property>
   </item>
   <item row="1" column="17">
    <property name="text">
     <string>I</string>
    </property>
   </item>
   <item row="1" column="18">
    <property name="text">
     <string>H</string>
    </property>
   </item>
   <item row="1" column="19">
    <property name="text">
     <string>G</string>
    </property>
   </item>
   <item row="1" column="20">
    <property name="text">
     <string>F</string>
    </property>
   </item>
   <item row="1" column="21">
    <property name="text">
     <string>E</string>
    </property>
   </item>
   <item row="1" column="22">
    <property name="text">
     <string>D</string>
    </property>
   </item>
   <item row="1" column="23">
    <property name="text">
     <string>C</string>
    </property>
   </item>
   <item row="1" column="24">
    <property name="text">
     <string>B</string>
    </property>
   </item>
   <item row="1" column="25">
    <property name="text">
     <string>A</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>420</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>420</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>420</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>561</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
